openstf_dbc/services/predictor.py

```diff
@@ -2,36 +2,137 @@
 #
 # SPDX-License-Identifier: MPL-2.0
 
+from enum import Enum
+from typing import List, Optional, Tuple, Union
+
+import pandas as pd
 from openstf_dbc.data_interface import _DataInterface
+from openstf_dbc.services.weather import Weather
+from openstf_dbc.utils.utils import get_datetime_index
+
+
+class PredictorGroups(Enum):
+    MARKET_DATA = "market_data"
+    WEATHER_DATA = "weather_data"
+    LOAD_PROFILES = "LOAD_PROFILES"
```

Comment:
minor remark: The switching between capital and small letters looks weird

Reply:
thx. I will change it.